src/pages/Assess.vue

```diff
@@ -197,8 +181,6 @@ export default {
             return this.displayPages[this.pageIdx - 1].items.length < 1;
         },
         movePage(forwards) {
-            if (this.proceedDialog()) return;
```

Comment:
Why has this been moved out of here? Doesn't it make more sense here?

Reply:
Line 180 and 200 were duplicating when using the next function.
I removed 200 because the move page function is also used by prior.
I;m not sure that its a good user experience for the user to be stuck on the page.
That being said, I've had to implement a proceed dialog temporarily for prior because this functionality hasn't been confirmed yet.